package.json

```diff
@@ -3,8 +3,8 @@
     "precommit": "lint-staged"
   },
   "lint-staged": {
-    "app/**/*.{js,scss}": [
-      "prettier --trailing-comma es5 --write",
+    "app/**/*.{js,jsx,scss}": [
+      "eslint --fix",
```

Comment:
`eslint --fix` does both:

1. linting (raises an error if something cannot be fixed automatically)
2. automatic formatting in place (using prettier)
3. write the fixes and formatted modifications to file automatically

This happens because we integrate eslint with prettier as per https://prettier.io/docs/en/integrating-with-linters.html#recommended-configuration.